src/sonic-config-engine/minigraph.py

```diff
@@ -2474,7 +2474,10 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
 
     # asymmetric PFC is disabled by default
     for port in ports.values():
-        port['pfc_asym'] = 'off'
+        if 'pfc_asym' not in port:
+            port['pfc_asym'] = 'off'
+        elif port['pfc_asym'] == '-':
+            del port['pfc_asym']
```

Comment:
@zhixzhu is this because some Cisco SKU has mix of ports supporting PFC asym and not?

Reply:
Some ports are not able to take pfc_asym config, even with pfc_asym 'off' config.

Reply:
Why? where do you see the issue with pfc_asym=off?

Reply:
Attached the email thread on the PIE port issue. https://github.com/user-attachments/files/18130067/Re.Code.drop.10.202405.0.11.Published.pdf
By default, all ports are configured pfc_asym off, and that caused orchagent cores.

Reply:
@zhixzhu Can we fix the orchagent code? Having 'off' and '-' for the same functionality may be confusing.

Reply:
For the orchagent crash, I can ignore the failure in setPortPfcAsym for cisco-8000.
https://github.com/sonic-net/sonic-swss/blob/master/orchagent/portsorch.cpp#L2075

Reply:
@zhixzhu Instead of checking the SKU, can we check the return status as NOT_SUPPORTED and skip?

Reply:
Yes, I can check if (status == SAI_STATUS_NOT_SUPPORTED), ignore failure.